docs/introduction.md

```diff
@@ -38,22 +42,14 @@ Running your own Registry is a great solution to integrate with and complement y
 
 It's also an essential component if you want to quickly deploy a new image over a large cluster of machines.
 
-Finally, it's the best way to distribute images inside an airgap environment.
-
+Finally, it's the best way to distribute images inside an air-gapped environment.
```

Comment:
Some people may not know what an air gap is. Maybe say "...inside an isolated network"